MAP/src/api/index.js

```diff
@@ -8,4 +10,36 @@ export const getCoronaDataApi = axios.create({
   baseURL: 'https://api.coronatracker.com/v5/analytics/trend',
 });
 
+
+export const getEnvironments = () => {
```

Comment:
is this used at all right now?? 
maybe you can add some more comments to the code explaining the current state of things here, and how this will work different in the future

Reply:
okay Noted